presto-orc/src/main/java/com/facebook/presto/orc/reader/SelectiveStreamReaders.java

```diff
@@ -38,6 +40,7 @@ private SelectiveStreamReaders() {}
     public static SelectiveStreamReader createStreamReader(
             StreamDescriptor streamDescriptor,
             Map<Subfield, TupleDomainFilter> filters,
+            List<OrcType> types,
```

Comment:
I think a better change would to to store OrcType (not just OrcTypeKind) in the StreamDescriptor.